src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Utils/Data+Compression.swift

```diff
@@ -0,0 +1,39 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import Compression
+
+extension Data {
+
+    func decompressZLib() throws -> Data {
+
+        // The maximum output size of the zlib decompression shall be set to 10 MB to protect against zip bomb attacks.
+        let tenMBCapacityLimitInByte = 10_000_000
```

Comment:
Is 10 * 1000 * 1000 correct or should it be 10 * 1024 * 1024 (10_485_760)?

Reply:
Right. I tried to sell you a hard drive. You got me.

Reply:
Wouldn't that be 'BiByte' then? 🤷‍♂️